src/router.jsx

```diff
@@ -94,9 +94,17 @@ const oldLinks = {
   mint: '/mint',
 };
 
+function WrappedRouter({ children }) {
+  return process.env.IPFS_DEPLOY ? (
+    <HashRouter history={history}>{children}</HashRouter>
```

Comment:
urls will work now?

Reply:
2 versions - Hash router for IPFS, and slash for server. After next.js we can avoid hashrouter